sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/README.md

```diff
@@ -1,22 +1,22 @@
 # Azure Event Hubs Checkpoint Store client library for .NET using Storage Blobs
 
-WORDS NEEDED...
+Intended as a companion to the `Azure.Messaging.EventHubs` client library, the Azure Event Hubs Blob Checkpoint Store enables using an Azure storage account as the durable persistence mechanism for an `EventProcessor`.  The constructs in this library plug into the `EventProcessor` allowing it to preserve its state, in the form of checkpoints, as a storage blob.   
 
-[Source code](.) | [Package (NuGet)](https://www.nuget.org/packages/Azure.Messaging.EventHubs.CheckpointStore.Blob/) | [API reference documentation](https://azure.github.io/azure-sdk-for-net/api/EventHubs/Azure.Messaging.EventHubs.CheckpointStore.Blob.html) | [Product documentation](https://docs.microsoft.com/en-us/azure/event-hubs/)
+[Source code](.) | [Package (NuGet)](https://www.nuget.org/packages/Azure.Messaging.EventHubs.CheckpointStore.Blob/) | [API reference documentation](https://azure.github.io/azure-sdk-for-net/api/Azure.Messaging.EventHubs.CheckpointStore.Blob.html) | [Product documentation](https://docs.microsoft.com/en-us/azure/event-hubs/)
```

Comment:
The docs haven't been generated yet because the package source doesn't exist yet.  Chicken-or-the-egg problem.   We need to get the package out and then I can follow-up on docs and make any tweaks here.

Reply:
That was a call made by the architecture board for the naming across languages.   Happy to bring it up for discussion, but I'll ask that be deferred until after this release.